ftest.sh

```diff
@@ -312,7 +312,8 @@ if ${SETUP_ONLY:-false}; then
 fi
 
 # now run it!
-export PYTHONPATH=./util:../../utils/py/:./util/apricot
+export PYTHONPATH=./util:../../utils/py/:./util/apricot:./install/lib/python2.7/site-packages:./install/lib/python3/site-packages
```

Comment:
(style)  line over 80 characters

Reply:
I'm not sure this will actually work for python3.  Won't a python3 script start to find stuff it thinks it wants in `./install/lib/python2.7/site-packages` and use it erroneously instead of the python3 equivalent?